topology/Massachusetts Green High Performance Computing Center/NRP/MGHPCCCachingInfrastructure.yaml

```diff
@@ -0,0 +1,23 @@
+Production: true
+SupportCenter: Community Support Center
+GroupDescription: Caching infrastructure in Nebraska
+GroupID: 1318
+
+Resources:
+  MGHPCC_NRP_OSDF_CACHE:
+    Active: false
+    Description: MGHPCC NRP OSDF Cache
+    ID: 1348
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Fabio Andrijauskas
+          ID: ef5d0be3ec1df913935853ed82396b1eb37a7db0
```

Comment:
We should have a backup.  How about Huijun?

Reply:
Added (in each new resource)